README.md

```diff
@@ -1,57 +1,16 @@
-# Northcoders House of Games API
+1. Copy Trello Board - Complete
```

Comment:
I'd remove the copy trello board step here - this readme is for people viewing your portfolio and wanting to snoop on this completed project so they just need the initial setup instructions